lib/loader/ast/instruction.cpp

```diff
@@ -998,7 +997,8 @@ Expect<void> Loader::checkInstrProposals(OpCode Code,
                              Offset, ASTNodeAttr::Instruction);
     }
   } else if (Code == OpCode::Return_call ||
-             Code == OpCode::Return_call_indirect) {
+             Code == OpCode::Return_call_indirect ||
+             Code == OpCode::Return_call_ref) {
```

Comment:
Not to add the `Return_call_ref` in this condition.
The `Return_call_ref` case will be blocked in this statement and always not run into the condition below.